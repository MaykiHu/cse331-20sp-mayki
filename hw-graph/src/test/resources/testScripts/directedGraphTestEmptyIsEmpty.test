#Create a graph
CreateGraph graph1

#Checks if empty, when it is empty
IsEmpty graph1