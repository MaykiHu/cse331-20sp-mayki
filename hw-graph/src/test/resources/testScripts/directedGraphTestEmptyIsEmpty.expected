#Create a graph
created graph graph1

#Checks if empty, when it is empty
graph1 is empty